### PR TITLE
Feature user signup

### PR DESCRIPTION
**Pull request by MartynaAnnaGottschling**

### CODE REVIEW

File: users/serializers.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
+from allauth.account import app_settings as allauth_settings
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import setup_user_email
+from allauth.utils import email_address_exists
+
 from django_countries.serializers import CountryFieldMixin
 from django_countries.serializer_fields import CountryField
+from django.utils.translation import ugettext_lazy as _
+
 from rest_framework import serializers
```
Comment (TheCM) [inline users/serializers.py]: You can import just needed fields.

File: users/serializers.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
+from allauth.account import app_settings as allauth_settings
+from allauth.account.adapter import get_adapter
```
Comment (TheCM) [inline users/serializers.py]: import as allauth_get_adapter

File: users/serializers.py (hunk 1 above)
Comment (MartynaAnnaGottschling) [reply users/serializers.py]: Done

File: users/serializers.py (hunk 2 above)
Comment (MartynaAnnaGottschling) [reply users/serializers.py]: Done

**Pull request merged**